### DISCUSSION TITLE
Link 와 router.push

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#42] [03.Technical]
@chojooyoung 그린도 같이 봐주시면 좋을 거 같아요🙂

<img width="782" alt="스크린샷 2022-07-31 오후 6 51 37" src="https://user-images.githubusercontent.com/72402747/182020760-b7c3e349-2ca7-49a5-b2d6-d3aad31fc3da.png">

[참고](https://github.com/prgrms-web-devcourse/Team-BackFro-ArtZip-FE/pull/38)

**Comment by chojooyoung** [comment]
롸...? 잘못 노티주신 것 같습니닷...!

**Reply by Hyevvy** [reply]
헉... 죄송합니다... ㄷ ㄷ...
